src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -21,32 +21,51 @@
 
 <template>
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
-		<div class="new-conversation"
-			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
+		<div class="new-conversation" :class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
 			<SearchBox v-model="searchText"
 				class="conversations-search"
 				:is-searching="isSearching"
 				@input="debounceFetchSearchResults"
 				@submit="onInputEnter"
 				@keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
-			<NewGroupConversation v-if="canStartConversations" />
+
+			<!-- Options -->
+			<NcActions>
+				<NcActionButton  @click="insertValue(t('spreed','is:unread'))" close-after-click>
+					<template #icon>
+						<MessageBadge :size="20" />
+					</template>
+					{{ t('spreed','Filter unread messages') }}
+				</NcActionButton>
+				<NcActionButton  @click="insertValue(t('spreed','is:mentioned'))" close-after-click>
+					<template #icon>
+						<AtIcon :size="20" />
+					</template>
+					{{ t('spreed','Filter unread mentions') }}
+				</NcActionButton>
+				<NcActionButton @click="renderNewGroupConversation" close-after-click>
+					<template #icon>
+						<Plus :size="20" />
```

Comment:
```suggestion
						<PlusIcon :size="20" />
```
Not critical, but we could import it as `PlusIcon` for icons names consistency